html/aboutMe.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="/Css/aboutMe.css">
+    <title>About Me</title>
+</head>
+
+<body>
+        <header>
+            <div class="navigation_bar">
+
+                <img src="/Img/Netflix.png" alt="" width="200" height="50">
+                <a href="/index.html">INICIO</a>
+                <a href="/html/history.html">HISTORY</a>
+                <a href="/html/contact.html">CONTACT</a>
+                <a href="/html/aboutMe.html">MY PROFILE</a>
+
+            </div>
+        </header>
+
+        <h1>About Me</h1>
+
+        <div class="aboutMe">
+            <div class="content">
+
+            
+            <img class="Image" src="/Img/image_Valen.png" alt="Image about me" width="300px" height="300px">
+            <p>I am 18 years old. I am a very shy person, many confuse that with being "believed" however it is a word
```

Comment:
open and close tags should ideally be in the same column, so it's easier to read